composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Minimal Application Factory for Mezzio",
     "type": "library",
     "require": {
-        "php": "8.1.*|8.2.*|8.3.*",
+        "php": "8.1.*|8.2.*|8.3.*|8.4.*",
```

Comment:
We could do with adding the new PHP versions to the CI matrix too, would you be able to add them there too please? We didn't adopt Laminas CI matrix on this yet; that would be a good addition in future though :100:

Reply:
Oops. Sure. Sorry for not thinking about that. Will sort out both points.

Reply:
Hey @asgrim. 👋🏽  If you're okay with the additional changes made, then could you merge the PR, pls?